Yosemite/YosemiteTests/PointOfSale/PointOfSaleItemMapperTests.swift

```diff
@@ -0,0 +1,147 @@
+import Testing
+import WooFoundation
+@testable import Yosemite
+
+struct PointOfSaleItemMapperTests {
```

Comment:
Nice having separate class with separate test cases! How about the scenario when more than one product/variation are passed even though the current use cases just pass one?

Reply:
[0dc409c](https://github.com/woocommerce/woocommerce-ios/pull/15710/commits/0dc409c31c54b1334281d54b404de81f3e34a328)